azure-pipelines.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     inputs:
       command: test
       projects: src/Steeltoe.All.sln
-      arguments: '--blame --no-build -c $(buildConfiguration) -v diag -f netcoreapp3.1 --filter "Category!=SkipOnLinux&Category!=Integration&FullyQualifiedName!~CircuitBreaker"'
+      arguments: '--blame --no-build -c $(buildConfiguration) -v n -f netcoreapp3.1 --filter "Category!=SkipOnLinux&Category!=Integration&FullyQualifiedName!~CircuitBreaker&FullyQualifiedName!~Steeltoe.Stream"'
```

Comment:
Why exclude these from this step? `&FullyQualifiedName!~Steeltoe.Stream`

Reply:
I was reverting to an earlier version - should have just reverted to the `main` version.